src/podofo/private/FreetypePrivate.cpp

```diff
@@ -87,12 +87,12 @@ FT_Face FT::CreateFaceFromBuffer(const bufferview& view, unsigned faceIndex,
             buffer = view;
 
         // Unconditionally re-create the face from the copied buffer
-        return createFaceFromBuffer(buffer, 0);
+        return createFaceFromBuffer(view, 0);
```

Comment:
I.e., if `tryExtractDataFromTTC` in line 86 worked, ignore that and use `view` instead anyway? Does that mean `tryExtractDataFromTTC` did something wrong?